### ISSUE TITLE
Code Coverage: Create tests for OrgPost.tsx

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#271]
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa-Admin code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `src/screens/OrgPost/OrgPost.tsx`

- When complete, all methods, classes and/or functions in the file will need to be tested.
- These tests must be placed in a single file with the name `src/screens/OrgPost/OrgPost.test.tsx`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- #241

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa-admin/tree/develop/src/screens/OrgPost/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- Create a code coverage account for your repo's preferred branch to generate the values when you do your commits for that branch to your repo.
    - _**NOTE:**_ Make sure you select the correct branch when you do the setup, or else the reporting will fail. You must also create a `CODECOV` GitHub secret for your repo as part of the process.
- The PR will show a report for the code coverage for the file you have added. You can also use that as a guide.

**Comment by priyang12** [comment]
hi can i work on this issue?

**Comment by SiddheshKukade** [comment]
@priyang12  if you are not working on this I would like to take this issue

**Comment by pinkman7009** [comment]
Is this issue still open? It's been a month. If no one is working on this, I would like to work on this. @palisadoes

**Comment by rv602** [comment]
@palisadoes as far as I remember, the codecov for ```OrgPost.tsx``` wasn't reaching up to 100% because of the error in tags attribute that occurs while organisation creation.

But that was raised as an issue by @VenomFate-619 and his PR - https://github.com/PalisadoesFoundation/talawa-api/pull/875 was also merged recently.
Hence I think that this issue is resolved and can be closed.

Currently I have my University's end semester examinations hence I will be able to verify this properly only after a few days, if necessary. Kindly excuse me for the same.
Thank you.

**Comment by noman2002** [comment]
> Is this issue still open? It's been a month. If no one is working on this, I would like to work on this. @palisadoes

Please check the current coverage of this file and let us know. If its below 100% I'll assign it to you.

**Comment by pinkman7009** [comment]
Sure, thanks!

**Comment by palisadoes** [comment]
I'm reassigning

**Comment by prathamesh-mutkure** [comment]
@palisadoes, can I take this up?

**Comment by noman2002** [comment]
@pinkman7009 Any update ? Its been more that 2 weeks.

**Comment by SiddheshKukade** [comment]
@noman2002 currently It has around 70% coverage. I can work on this. please assign

**Comment by palisadoes** [comment]
Fix deprecated `get-changed-files` Pull Request GitHub Action

**Comment by palisadoes** [comment]
Unassigning due to inactivity. We need to give others a chance.

**Comment by adi-uchiha** [comment]
Can I work on this ?

**Comment by violetakpeh** [comment]
@palisadoes Kindly assign me this issue

**Comment by adi-uchiha** [comment]
Sorry for the delay I will submit a PR in before 9PM (IST)
